src/OAuth1.php

```diff
@@ -78,12 +78,6 @@ public function buildAuthUrl(
         array $params = []
     ): string {
         $requestToken = $this->fetchRequestToken($incomingRequest);
-        if (!is_object($requestToken)) {
```

Comment:
Why was it removed?

Reply:
It was moved into fetchRequestToken, because this method can only return OAuthToken

Reply:
I don't see `$this->getState('requestToken');` in `fetchRequestToken`.